libs/shared/src/containers/TransactionList/index.tsx

```diff
@@ -28,9 +51,10 @@ function buildPagination(transactions: Transaction[], maxRowCount: number, curre
     const startRow = currentPage * maxRowCount;
     transactions = transactions.slice(startRow, startRow + maxRowCount);
 
-    return [transactions, (<Table.Footer>
-      <Table.Row>
-        <Table.HeaderCell colSpan='4'>
+    return [transactions, (
+    <Grid>
```

Comment:
Switched everything to grids: it's way easier to work with.

Reply:
I think between Grid & Flex display, there isn't really much need for Tables anymore.  Old habits die hard though!

Reply:
Tables are bad and evil. Tables are bad and evil.